Miguel9r/CS179J-Team10
# Ble service branch merge into Master (#14)

* updated BLE services to look for specific BLE module service and characteristics

* added more to UI components and changed UUID strings to match ble module arduino strings

* starting bluetooth service from scratch

* displaying paired devices on android app in ListView

* detecting all device names

* uploading changes to finding paired devices

* listing out device addresses on terminal

* trying to establish a connection between module and app

* trying to establish a bluetooth connection

* establishing ble connection

* pushing changes to establish write and read data

* added visibility for LEDButton

* set enabled

* establishing write bluetooth functionality

* establishing writing to module

* different changes to writing data'

* changed visability

* uploading more testing on ble communication

* Debugging.

* Debugging more.

* Update MainActivity.java

* casting byte value

* Updating

* Debugging app code

* int value being passed with .write()

* pushing up decimal values

* Debugged Arduino and Android code to turn on LED button

* Create bluetooth_test.ino

simplified bluetooth testing arduino file

* uploading changes to flash button functionality

* fixed debugging messages

* Update MainActivity.java

Added a ton of changes to account for all the different buttons.

* tested more button implementation

* Update MainActivity.java

Adding debugging statements.

* removed unnecessary code -- accidentally removed menu description text

* refactored directories and changed the landing icon on whatever device it is used on

* uploading UI changes to buttons

* added some functionality for the left and right buttons

* improved functionality for buttons and mainly for center button to change tabs-wise

* added splash screen

* flash and camera button should have functionality to be received by the hardware

* deleting arduino sample dir. and uploading commands list

* deleting loop that printed commands index

* refactored dir name

* deleting paired list after connect button

* changed the camera area size and added the state the app is controlling

* Shifting message around.

* uploading changes to read() for arduino file

* quick arduino test signal program

* Modifying entire Mobile App GUI.

Moved some items around, changed colors, added proper padding between buttons, centered some buttons and text. Overall cleanup of the app GUI.

* Update SplashScreen.java

Added delay to splash screen so that Jesus can show off his splash screen :)

* Shifting more app stuff around, and added static state for buttons.

* Stuff.

* Update MainActivity.java

* Update activity_main.xml

* fixed up UI to display correct state

* Update MainActivity.java

Co-authored-by: Miguel Rojas <mroja008@ucr.edu>
Co-authored-by: klabo001 <klabog001@gmail.com>
Co-authored-by: byu014 <byu014@ucr.edu>
